src/app/public/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -85,4 +88,20 @@ export class SkyAgGridWrapperComponent implements AfterContentInit {
       }
     }
   }
+
+  private get isInEditMode(): boolean {
+    const primaryGridEditing = this.agGrid && this.agGrid.api && this.agGrid.api.getEditingCells().length > 0;
+    if (primaryGridEditing) {
+      return true;
+    } else {
+      let innerEditing: boolean = false;
+      this.agGrid.api.forEachDetailGridInfo((detailGrid: DetailGridInfo) => {
```

Comment:
Looking at the logic branches, `this.agGrid` can still be undefined here and cause an error. Is there a better way to organize these conditionals to check for that?

Reply:
I moved the checks (which are really sanity checks) just to give us strict null checks here.